projects/ngx-vest-forms/src/lib/directives/validation-options.directive.ts

```diff
@@ -0,0 +1,25 @@
+import { Directive, input } from '@angular/core';
+
+/**
+ * Define validation options for a form, form model group or form model
+ */
+@Directive({
+  selector: '[validationOptions]',
```

Comment:
we use the sc prefix. For element components we use:
`<sc-my-component></sc-my-component>`

For attribute components we use:
`<form sc-my-form></form>`

For directives we use camelcase:
`<form scMyForm></form>`

Reply:
I just come to the realisation that the `validateRootForm` is the same, so I violated my own rules.
WDYT? I prefer not to do a breaking change.

Reply:
I removed the directive and replaced it with inputs.

For the `validateRootForm` I would introduce a second selector (`scValidateRootForm`) and deprecate the old one.
The old one could then be removed with next major or minor version.